parquet/src/main/scala/magnolify/parquet/MagnolifyParquetProperties.scala

```diff
@@ -30,6 +30,27 @@ trait MagnolifyParquetProperties extends Serializable {
 object MagnolifyParquetProperties {
   val Default: MagnolifyParquetProperties = new MagnolifyParquetProperties {}
 
-  val WriteAvroCompatibleArrays: String = "magnolify.parquet.write-grouped-arrays"
+  val WriteArrayEncoding: String = "magnolify.parquet.write-array-encoding"
+  val Ungrouped: String = "ungrouped"
+  val OldArrayEncoding: String = "old-array-encoding"
+  val NewListEncoding: String = "new-list-encoding"
+
   val WriteAvroSchemaToMetadata: String = "magnolify.parquet.write-avro-schema"
 }
+
+sealed trait ArrayEncoding
+
+object ArrayEncoding {
+  case object Ungrouped extends ArrayEncoding
+  case object OldArrayEncoding extends ArrayEncoding
+  case object NewListEncoding extends ArrayEncoding
```

Comment:
Not sure if I 100% like the naming conventions on these. In parquet-avro the equivalent property is called [parquet.avro.write-old-list-structure](https://github.com/apache/parquet-java/blob/apache-parquet-1.15.2/parquet-avro/src/main/java/org/apache/parquet/avro/AvroWriteSupport.java#L69)